src/SingleCellAnnotator.imjoy.html

```diff
@@ -105,7 +105,13 @@
   label_display_mode: null,
   uri: null,
   location: null,
-  location_selected: null
+  discarded: null
```

Comment:
this is used in toggleDiscard, right? @oeway

Reply:
True, yes, sorry, I thought it was in `data`, but the default value should be `false` then.

Reply:
set